Makefile

```diff
@@ -82,6 +82,11 @@ update: | update-common
 
 # a phony target, because teaching `make` how to do conditional recompilation of Nim projects is too complicated
 
+# Whisper tests
+testwhisper: | build deps
```

Comment:
I'm assuming this is purposefully left out of the `test` target?

Reply:
Yeah don't really see a need to have it there, will just prolong CI. It passes locally now and is basically identical to waku1, which is what we care about here I think